src/server/pkg/storage/chunker/chunker_test.go

```diff
@@ -0,0 +1,57 @@
+package chunker
+
+import (
+	"bytes"
+	"io"
+	"math/rand"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+)
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+// generates random sequence of data (n is number of bytes)
+func randSeq(n int) []byte {
+	b := make([]rune, n)
+	for i := range b {
+		b[i] = letters[rand.Intn(len(letters))]
+	}
+	return []byte(string(b))
+}
```

Comment:
I've actually implemented a similar function a few times in our code base. I don't think any are still around. I recently discovered there's a random data reader in the crypto package: https://golang.org/pkg/crypto/rand/ and have been using that since.

Reply:
That is useful, I will probably switch to that.